src/System.Memory/src/System.Memory.csproj

```diff
@@ -27,12 +24,22 @@
     <Compile Include="System\SpanHelpers.Clear.cs" />
     <Compile Include="System\ThrowHelper.cs" />
   </ItemGroup>
-  <ItemGroup>
+  <ItemGroup Condition="'$(IsPartialFacadeAssembly)' != 'true'">
```

Comment:
Could we just remove the netstandard1.0 build in Configuration.props, and remove any files that are specific to that build? @joperezr, is there any reason to build for older frameworks anymore?

If we should continue to build for netstandard1.0, I suggest using a condition based on the TargetGroup so that it's clear which runtime which items are for.